src/org/isf/admission/test/Tests.java

```diff
@@ -354,8 +354,8 @@ public void testIoNewAdmission()
 			
 			Admission admission = testAdmission.setup(ward, patient, admissionType, diseaseIn, diseaseOut1, 
 					diseaseOut2, diseaseOut3, operation, dischargeType, pregTreatmentType, 
-					deliveryType, deliveryResult, true);;
-					result = admissionIoOperation.newAdmission(admission);
+					deliveryType, deliveryResult, true);
+            result = admissionIoOperation.newAdmission(admission);
```

Comment:
is the conseguent indentation correct? tabs have been converted into spaces

Reply:
I see, so you should also have noticed a number of warnings we could get rid of.

Anyway, is the above conseguent indentation correct? Tabulations have been converted into spaces maybe